bin/deploy_frontend/common_functions.sh

```diff
@@ -29,7 +29,7 @@ export PATH="$PATH:/usr/bin:/usr/local/bin"
 DEBUG=true
 
 # Number of times to check for a resouce to be in the desired state.
-WAITER_ATTEMPTS=60
+WAITER_ATTEMPTS=180
```

Comment:
mostly indifferent between the two. the only (mostly-insignificant) advantage is that the alternative approach would complete a second or two slower, on average.